tests/runtime/kubernetes/test_kubernetes_executor.py

```diff
@@ -66,6 +64,10 @@ def setUp(self):
     def tearDown(self):
         self.session.close()
 
+    @classmethod
+    def tearDownClass(cls) -> None:
+        dump_kubernetes_logs()
```

Comment:
Wouldn't it be better to use `tearDown`? Then we will achieve more information (and more noise). 
Edit: maybe we can use this feature? https://docs.pytest.org/en/latest/example/simple.html#post-process-test-reports-failures

Reply:
Thought about it but it's better to do this at the end as usually when the first test fails, the rest will as well due to kubernetes misconfiguration. This is really here in order to see that kubernetes was misconfigured, so I prefer to see all the logs after all the tests complete.

Reply:
And since we are dumping all logs from kubernetes this would dump anyway all the logs from all tests so far, so it makes little sense really.

Reply:
I also thought about using post-process but I only want to run it only when the kubernetes tests are run - I prefer to do it in the tearDownClass for now rather than checking additionally in the post_process hook that kubernetes env is set....  But maybe that makes more sense indeed. I will think about it.

Reply:
I chaged it completely now. I moved all the extract/upload to the entrypoint_ci.sh bash script - it makes much more sense there (we have guarante that the output containing files.io link) will be printed in the log from Travis